inductiva/resources/storage.py

```diff
@@ -0,0 +1,16 @@
+"""Methods to interact with the user storage resources."""
+from absl import logging
+import inductiva
+from inductiva.client.apis.tags import instance_api
+
+
+def get_used():
```

Comment:
get_used_storage_disk()?

Reply:
Since it is already under `inductiva.resources.storage` I don't think there is need to mention storage again in the function name. Disk is a little misleading. Maybe get_space_used?

Reply:
Yes, you are correct. I thought you wanted to do: `inductiva.resources.get_storage_space_used()`.